coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -215,11 +244,6 @@ def __call__(self, message: fedora_messaging.api.Message):
         #logger.debug(message.topic)
```

Comment:
We should clean up these commented debug lines.

Reply:
I chose to leave them commented out in a previous commit: see https://github.com/coreos/fedora-coreos-releng-automation/pull/22#discussion_r313078517

Reply:
Since we're bumping the logger level to `INFO` in a later commit is there any reason not to just uncomment them or to change up the comments to describe the fields/variables available on the object rather than just commented out logger lines?

Reply:
since we did change to INFO, I agree. I'll uncomment them in that commit